localstack/aws/handlers/cors.py

```diff
@@ -88,9 +111,8 @@ def _get_allowed_cors_origins() -> List[str]:
     ]
     # Add allowed origins for localhost domains, using different protocol/port combinations.
     # If a different port is configured for EDGE_PORT_HTTP, add it to allowed origins as well
-    _ports = set([config.EDGE_PORT] + ([config.EDGE_PORT_HTTP] if config.EDGE_PORT_HTTP else []))
     for protocol in {"http", "https"}:
-        for port in _ports:
+        for port in _get_allowed_cors_ports():
```

Comment:
That's what I thought and did at first, but then it would be generated only once, and in your other tests where you monkey patch `ALLOWED_CORS_ORIGINS`, you'd have to monkey patch `_ALLOWED_INTERNAL_PORTS` before. I thought it'd make your tests a bit more complicated.

https://github.com/localstack/localstack/blob/be7178da606ec4b15b98bd287fc339ad0c870d0d/tests/unit/test_cors.py#L18-L21